Wrap up selectors

tkdrob opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Breaking change
<!--
  If your PR contains a breaking change for existing users, it is important
  to tell them what breaks, how to make it work again and why we did this.
  This piece of text is published with the release notes, so it helps if you
  write it towards our users, not us.
  Note: Remove this section if this PR is NOT a breaking change.
-->


## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->
Now that all services use selectors, this PR proposes removing service call examples for items that use any selector except for `text:` and `object:`.
Removed all instances of `mode: slider` and `step: 1` as these are the default
Some grammar fixes
Descriptions clean up as the user now sees described options via the selectors
Added some forgotten selectors
Normalized order of keys and options

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [ ] Untested files have been added to `.coveragerc`.

The integration reached or maintains the following [Integration Quality Scale][quality-scale]:
<!--
  The Integration Quality Scale scores an integration on the code quality
  and user experience. Each level of the quality scale consists of a list
  of requirements. We highly recommend getting your integration scored!
-->

- [ ] No score or internal
- [ ] 🥈 Silver
- [ ] 🥇 Gold
- [ ] 🏆 Platinum

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html
[quality-scale]: https://developers.home-assistant.io/docs/en/next/integration_quality_scale_index.html
[docs-repository]: https://github.com/home-assistant/home-assistant.io

tkdrob committed:
Wrap up selectors

tkdrob committed:
Wrap up selectors
Rebase

frenck commented:
Rebased to deal with CI failures from yesterday's GitHub Outage

frenck commented on homeassistant/components/light/services.yaml:
```diff
@@ -9,15 +9,12 @@ turn_on:
           max: 300
-          step: 1
           unit_of_measurement: seconds
-          mode: slider
```
Something is off with the default.

According to the schema:

The default is `slider`:

```py
vol.Optional(CONF_MODE, default="slider"): vol.In(["box", "slider"])
```

however, the frontend doesn't seem to care:

![image](https://user-images.githubusercontent.com/195327/118458450-0dd45d80-b6fb-11eb-82fd-165b009a2bcc.png)

tkdrob replied:
Hm. The step is ignored, too.

frenck replied:
I've reached out to frontend to see what is going on.

frenck replied:
Fixed in home-assistant/frontend#9190

frenck committed:
Upgrade aiodns to 3.0.0 (#50712)

frenck committed:
Remove side effects from Watson TTS init (#50716)

Noltari committed:
Update AEMET library to latest version (#50222)

Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>

mountainsandcode committed:
Add control of hardware buttons to Sonos (#49977)

mib1185 committed:
Add strict type annotations to ampio (#50699)

farmio committed:
Update xknx to version 0.18.2 (#50491)

* xknx 0.18.2

* individual colors without switch

* make `setpoint_shift_mode` optional

* Update homeassistant/components/knx/schema.py

janiversen committed:
Simplify calls to pymodbus (#50717)

* simplify pymodbus_call.

Do not call with a function object and a check attribute, call instead
with CALL_TYPE*.

Avoid if <type> call x else call y.
Call async_pymodbus_call directly, instead of unpacking/packing.

* Declare call type in __init__.

* Modbus.py back to 100% test coverage.

frenck committed:
Upgrade pyupgrade to v2.16.0 (#50756)

frenck committed:
Remove discovery from iCloud (#50751)

farmio committed:
Create KNX cover entities directly from config (#50707)

CantankerousBullMoose committed:
Rescan static wemo (#49934)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>
Co-authored-by: Franck Nijhof <git@frenck.dev>

frenck committed:
Upgrade geopy to 2.1.0 (#50714)

emontnemery committed:
Refactor MQTT basic light pt1: Add add_topic helper (#50759)

austinmroczek committed:
Fix armed_night logic in totalconnect alarm and add tests (#50694)

* Fix armed_night, add tests for alarm

* end assertions with expected values

emontnemery committed:
Hoist ATTR_LAST_RESET from utility_meter to SensorEntity (#50757)

frenck committed:
Extend targets for entity component services (#50760)

chemelli74 committed:
Fix strings for UPNP (#50762)

mib1185 committed:
Add strict type annotations to amazon_polly (#50697)

* add strict type annotations

* apply suggestions

Co-authored-by: Erik Montnemery <erik@montnemery.com>

nikito7 committed:
Change Modbus error message to bytes (#50725)

bieniu committed:
Allow some failures before setting Xiaomi Miio MIOT air purifiers unavailable (#50755)

janiversen committed:
Fix missing await in modbus platforms (followup on async PR) (#50710)

MartinHjelmare committed:
Clean up smhi redundant code (#50765)

frenck commented:
This PR needs a rebase, because of a change we did to remove some "magic" from the frontend.

tkdrob committed:
Merge branch 'dev' of https://github.com/home-assistant/core into selectors

tkdrob committed:
Adjust some descriptions and examples

frenck commented:
@tkdrob Uh? 😬

tkdrob commented:
> @tkdrob Uh? 😬 

Shall I reopen in another? This is a mess
